sql/step_91_add_dicom_data_table.sql

```diff
@@ -0,0 +1,30 @@
+-- add foreign key from dicom to dicom_data
+-- this means that the dicom table will be responsible in maintaining the relationship between dicom and the dicom data table
+ALTER TABLE OH_DICOM ADD COLUMN DMD_DATA_ID int(11);
```

Comment:
in the OH_DICOM table prefixes are all DM_

Reply:
changed